code/datums/gamemodes/revolution.dm

```diff
@@ -17,6 +17,8 @@
 	var/list/datum/mind/revolutionaries = list()
 	var/finished = 0
 	var/const/waittime_l = 600 //lower bound on time before intercept arrives (in tenths of seconds)
+	var/const/TrackerTime_min = 27 MINUTES //upper bound on time before intercept arrives (in tenths of seconds)
+	var/const/TrackerTime_max = 30 MINUTES //lower bound on time before intercept arrives (in tenths of seconds)
```

Comment:
Incorrect comments, also looks like you placed it between two corresponding variables.